.github/workflows/docs.yml

```diff
@@ -30,6 +30,8 @@ jobs:
         run: |
           python -W ignore -m pip install --upgrade pip
           python -W ignore -m pip install -r requirements-all.txt
+      - name: Test autogeneration of admonitions
+        run: pytest -v --tb=short tests/docs/admonition_inserter.py
```

Comment:
what's the difference between using `--tb=short` and not using it?

Reply:
The default `auto` mode provides long traceback for the 1st and last entry, and short traceback for other entries ([docs](https://docs.pytest.org/en/6.2.x/usage.html#modifying-python-traceback-printing)). I'm requesting short traceback for all entries. I made sure my exception messages are informative enough.

Reply:
What does 1st and last entry mean here? Do you have an example at hand for comparision?

Reply:
I'm not sure entirely myself, my guess is the entries in a call stack.
[This is the result](https://pastebin.com/5fe6a8Ah) with no `--tb=short` (=`--tb=auto`).
[This is the result](https://pastebin.com/wr4WjSJj) with `--tb=short`.

Reply:
Okay, that looks sensible to me :D